recipes/echtvar/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.2.2" %}
+
+package:
+  name: echtvar
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true  # [osx]
```

Comment:
```suggestion
```
I had no trouble getting this to build on Intel macOS - feel free to check the recipe I linked in my other comment to see how it looked.